deel-3-webpagina/index.html

```diff
@@ -7,10 +7,30 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
+<header>
+    <h3 class="header-h3">ANALYTICA</h3>
```

Comment:
Het is beter om dit alleen met hoofdletter A te schrijven en dan later in css de text-transform property te gebruiken om de letters te transformeren naar uppercase